test/unit/cart-button.ts

```diff
@@ -0,0 +1,110 @@
+import CartButton from '../../src/cart-button';
+import suite from './_suite';
+
+suite('CartButton', ({ expect, stub, spy }) => {
+  let cartButton: CartButton;
+
+  beforeEach(() => cartButton = new CartButton());
+
```

Comment:
Add check for state you expect from default values.
ie, like this: https://github.com/groupby/storefront-breadcrumbs/blob/master/test/unit/breadcrumbs.ts#L28